### PR TITLE
add benchmarks of UPDATE perf in transactions

### PR DESCRIPTION
**Pull request by dt**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/3940)

### CODE REVIEW

**Comment by tschottdorf** [comment]
could you add results to commit/pr message?

File: sql/bench_test.go [hunk 1]
```diff
@@ -281,6 +334,22 @@ func BenchmarkUpdate_Postgres(b *testing.B) {
...
 
+func BenchmarkUpdateTxn_Cockroach(b *testing.B) {
+	benchmarkCockroach(b, runBenchmarkUpdateTxn)
+}
+
+func BenchmarkUpdateTxn_Postgres(b *testing.B) {
+	benchmarkPostgres(b, runBenchmarkUpdateTxn)
+}
+
+func BenchmarkUpdateTwoRowsTxn_Cockroach(b *testing.B) {
```
Comment (petermattis) [inline sql/bench_test.go]: In other benchmarks I've used numbers in the benchmark name instead of spelling them out: `s/Two/2/g`.

**Comment by petermattis** [comment]
LGTM, though I'd like to see the results in the commit message as well.

**Comment by dt** [comment]
Added.

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/3940#-:-K8bwfmAbFOAbMdRic6Y:1610000531)*

**Comment by dt** [comment]
Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---

*[sql/bench_test.go, line 345 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/3940#-K8Zqnuv9R4oOXtNmu9B-r1-345:-K8bwhLW7P_0cgPKMIrA:-499146009) ([raw file](https://github.com/cockroachdb/cockroach/blob/4241a92ceded36227ed2d20cfabf5f7019290f08/sql/bench_test.go#L345)):*
Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/3940)*

**Pull request merged**